cluster/gce/gci/configure.sh

```diff
@@ -171,18 +171,19 @@ function install-node-problem-detector {
 
 function install-cni-binaries {
   #TODO(andyzheng0831): We should make the cni version number as a k8s env variable.
-  local -r cni_tar="cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz"
-  local -r cni_sha1="1d9788b0f5420e1a219aad2cb8681823fc515e7c"
+  local -r cni_tar="cni-plugins-amd64-v0.6.0.tgz"
+  local -r cni_version="v0.6.0"
+  local -r cni_sha1="d595d3ded6499a64e8dac02466e2f5f2ce257c9f"
   if is-preloaded "${cni_tar}" "${cni_sha1}"; then
     echo "${cni_tar} is preloaded."
     return
   fi
 
   echo "Downloading cni binaries"
-  download-or-bust "${cni_sha1}" "https://storage.googleapis.com/kubernetes-release/network-plugins/${cni_tar}"
+  download-or-bust "${cni_sha1}" "https://github.com/containernetworking/plugins/releases/download/${cni_version}/${cni_tar}"
```

Comment:
This has to be downloaded from GCS not Github.

@freehan -- let's upload a CNI 0.60 tarball to GCS.

Reply:
I will take this up this week.

Reply:
cc @kubernetes/sig-release-feature-requests @timothysc @jdumars 
Should this really be downloaded from GCS? We should make sure the content there is transparently pushed / handled, find a process for it, make it not Googler-only pushable, make sure checksums are used for verification, etc.

I don't understand why we need to push this to GCS anymore? Earlier we had to do it for a) we used a "random" git sha and b) release binaries were missing for some architectures.

Those two issues are now fixed. I think we should use Github if just possible.

Reply:
The CNI download here is run only for the GCE environment, it would be better to avoid cross dependencies with Github and depend only on GCE itself (i.e. GCS). For example, if Github went down, then it would take out GCE-based clusters.

Reply:
Ok, earlier the GCS builds have been used everywhere else as well. Anyway, let's just move the Github-released artifacts then, not reinvent the releasing scripts and build things yourselves.
It should be a 1:1 mirror of Github then.

Reply:
And provide visibility to others for this process.

Reply:
Uploaded CNI tarball to GCS:
link: 
https://storage.googleapis.com/kubernetes-release/network-plugins/cni-plugins-amd64-v0.6.0.tgz

Reply:
I think your suggestion that will depend on a change in the CNI repository? We can move ahead with this fix in the mean time while that is being resolved?

Reply:
> I don't understand why we need to push this to GCS anymore? Earlier we had to do it for a) we used a "random" git sha and b) release binaries were missing for some architectures.

> Those two issues are now fixed. I think we should use Github if just possible.

@luxas Agree. No need to have another copy for GCS. @bowei We did this earlier because CNI didn't provide binary artifacts for all platforms that kubernetes supports. So we build arch-specific artifacts. Since v0.5.2, they do.

Reply:
`cluster/gce/**` is only applicable to GCE environment. I think generically your comment is true, but not for the GCE-related cluster configuration scripts.